testing/src/main/java/io/stargate/it/http/restapi/RestApiTestBase.java

```diff
@@ -0,0 +1,148 @@
+package io.stargate.it.http.restapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.auth.model.AuthTokenResponse;
+import io.stargate.it.BaseIntegrationTest;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.http.models.Credentials;
+import io.stargate.it.storage.StargateConnectionInfo;
+import io.stargate.web.restapi.models.ColumnDefinition;
+import io.stargate.web.restapi.models.GetResponseWrapper;
+import io.stargate.web.restapi.models.PrimaryKey;
+import io.stargate.web.restapi.models.TableAdd;
+import io.stargate.web.restapi.models.TableResponse;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+
+/** Intermediate IT base class for use by REST API integration tests. */
+abstract class RestApiTestBase extends BaseIntegrationTest {
```

Comment:
Just FYI, this name may create some confusion upon merging into the v2.0.0 package, where a `io.stargate.it.http.BaseRestApiTest` has been added already.

Reply:
Ohhh. Thanks, I'll change that then!

Reply:
Ok I think I'll get this merged for `master`, merge & refactor in v2.0.0 to combine classes.

Reply:
I would say this functionality already exists, check this peace of code :arrow_down:  https://github.com/stargate/stargate/blob/d0d0ddf15f05c9185d3e0b6ac9a0d3df4cb32ae7/testing/src/main/java/io/stargate/it/http/docsapi/CollectionsResourceIntTest.java#L83-L94

So `@CqlSessionSpec` and `@TestKeyspace CqlIdentifier keyspace` do the trick.. I like this annotation approach way more..

Reply:
And I am not convinced create table and insert rows should be done is such a way.. The `CqlSessionSpec` has `initQueries` properties that can create table easily or add data transparently for the test setup..